### PR TITLE
Feature/108 jandj part1

### PR DESCRIPTION
**Pull request by e-lipman**
feat: Update vaccine results text (for typos and for single dose vaccines). Add some ui/server logic for single dose vaccines.

### CODE REVIEW

File: app/ui.R [hunk 1]
```diff
@@ -95,11 +95,15 @@ ui <- fluidPage(
             radioButtons('vaccine', "Which of the available COVID-19 vaccines did you recieve?",
                          choiceNames = unname(vaccine_labels), choiceValues = names(vaccine_labels),
                          inline=TRUE),
-            selectInput('doses', "How many doses of this vaccine have you recieved?",
-                         1:2),
+            conditionalPanel(
+              condition = "input.vaccine != 'johnsonandjohnson'",
+              selectInput('doses', "How many doses of this vaccine have you recieved?",
```
Comment (xindyhu) [inline app/ui.R:100]: "recieved" should be changed to "received"

File: app/ui.R [hunk 2]
```diff
@@ -95,11 +95,15 @@ ui <- fluidPage(
...
+                           1:2),
+              ),
             sliderInput("days_since_last_dose", 
                         HTML("How many days ago did you recieve your most recent dose? <sub class = 'text-info'>If more than 14 days ago, select 14.</sub>"),
```
Comment (xindyhu) [inline app/ui.R:104]: receive

File: app/ui.R [hunk 3]
```diff
@@ -95,11 +95,15 @@ ui <- fluidPage(
             radioButtons('vaccine', "Which of the available COVID-19 vaccines did you recieve?",
```
Comment (xindyhu) [inline app/ui.R:95]: receive

**Review by xindyhu — CHANGES_REQUESTED** [review]

File: app/ui.R (hunk 1 above)
Comment (e-lipman) [reply app/ui.R:100]: Thank you for catching!

**Review by xindyhu — APPROVED** [review]

**Pull request merged**